azure-iot-device/tests/common/pipeline/test_pipeline_stages_mqtt.py

```diff
@@ -388,33 +361,39 @@ def test_provider_function_throws_exception_2(
         assert handler_before == handler_after
 
 
-@pytest.mark.describe("MQTT Provider pipeline stage events from transport library")
+@pytest.mark.describe("Provider - .on_mqtt_message-received()")
 class TestMQTTProviderTransportEvents(object):
-    @pytest.mark.it("fires an IncomingMessage event for each mqtt message received")
+    @pytest.mark.it("Fires an IncomingMessage event for each mqtt message received")
     def test_incoming_message_handler(self, stage, create_provider, mocker):
         stage.provider.on_mqtt_message_received(topic=fake_topic, payload=fake_payload)
         assert stage.previous.handle_pipeline_event.call_count == 1
         call_arg = stage.previous.handle_pipeline_event.call_args[0][0]
         assert isinstance(call_arg, pipeline_events_mqtt.IncomingMessage)
 
-    @pytest.mark.it("passes topic and payload as part of the IncomingMessage event")
+    @pytest.mark.it("Passes topic and payload as part of the IncomingMessage event")
```

Comment:
>topic [](start = 28, length = 5)

So here and everywhere I guess, see that other comment about checking args passed should probably be implicit in checking the call to the function/method, rather than it's own separate test (unless you think there's a really good reason for it to be separate)